src/main/java/com/github/lunatrius/laserlevel/client/gui/marker/GuiMarkers.java

```diff
@@ -61,6 +61,7 @@ protected void actionPerformed(final GuiButton guiButton) {
         if (guiButton.enabled) {
             if (guiButton.id == this.btnAdd.id) {
                 final Marker marker = new Marker(new BlockPos(this.mc.thePlayer), 1, 0x0000BF);
+		marker.maxdistance = 64;
```

Comment:
Can be moved into the `Marker` constructor as it's supposed to be the default value for all objects.